frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/vm-settings-tab.tsx

```diff
@@ -102,9 +103,10 @@ export const VMSettingsTabComponent: React.FC<VMSettingsTabComponentProps> = ({
   );
 
   React.useEffect(() => {
-    if (!isDevicesInitialized) {
-      const templateDevices = toShallowJS(iUserTemplate)?.data?.objects?.[0]?.spec?.template?.spec
-        ?.domain?.devices;
+    const iUserTemplateSimplified = iUserTemplate?.toJS();
```

Comment:
```suggestion
    const template = iUserTemplate?.toJS();
```
It's simplified by converting immutable to regular object, so if you simplify the object to be a regular JS object, you can also drop the `i` from the name (it's not immutable anymore) the `user` because it's just a template, so you are left with just template.

Reply:
btw/ maybe you need only the `data` to get the `template` ?